Epam.TestAutomation.Tests/CareersPageTests.cs

```diff
@@ -37,20 +40,130 @@ public void HoverOverCareerMenuTest()
     }
 
     [Test]
-    public void DDTKeyWordPhrases()
+    [TestCaseSource(nameof(GetSearchTestData1))]
+    public void DDTKeyWordPhrases(SearchModel searchModel)
     {
         action.MoveToElement(BrowserFactory.Browser.FindElement(careersPage.careersButton)).Build().Perform();
```

Comment:
You click to the Careers button from mainPage. So change careersPage.careersButton to the mainPage.careersButton.

Reply:
And the same for JobListingButton. Change from careersPage.JobListingsButton to mainPage.JobListingsButton.